deis/azuredeploy.json

```diff
@@ -0,0 +1,371 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2014-04-01-preview/deploymentTemplate.json",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "newStorageAccountName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the of the storage account for VM OS Disk"
+            }
+        },
+        "vmNamePrefix": {
+            "type": "string",
+            "defaultValue": "coreos",
+            "metadata": {
+                "description": "Prefix for the names of the VMs"
+            }
+        },
+        "virtualNetworkName": {

```

Comment:
This can become a variable and one less parameter for the user to enter since it is only RG unique